app/src/main/java/com/github/se/project/model/profile/ListProfilesViewModel.kt

```diff
@@ -104,7 +104,7 @@ open class ListProfilesViewModel(private val repository: ProfilesRepository) : V
    *
    * @param profile The Profile document to be selected.
    */
-  fun setCurrentProfile(profile: Profile) {
+  fun setCurrentProfile(profile: Profile?) {
```

Comment:
I don't really see why this is necessary, but if it helps the testing it shouldn't be harmful either